hw/ip/adc_ctrl/doc/theory_of_operation.md

```diff
@@ -49,6 +49,10 @@ Note that the time taken in this step depends on the properties of the ADC.
 
 8. *Wait sleep time*: The controller will wait for the next sample timer to time out before restarting at step (1).
 
+If the controller is configured as oneshot mode, ([`adc_en_ctl.oneshot_mode`](../data/adc_ctrl.hjson#adc_en_ctl_1)),
+the controller waits for high value (1) from channel 0 and 1 in order without evaluatging filters,
+and it goes back to power off state after setting [`adc_intr_status.oneshot`](../data/adc_ctrl.hjson#adc_intr_status_1).
```

Comment:
Nit alert: 
- we typically use one sentence per line for markdown files.
- typo: evaluatging -> evaluating the

Reply:
updated.